packages/api/tests/endpoints/test-s3credentials.js

```diff
@@ -6,70 +6,86 @@ const request = require('supertest');
 
 const {
   testUtils: {
-    randomString
+    randomId
   },
   aws: {
     lambda
   }
 } = require('@cumulus/common');
-const { verifyJwtToken } = require('../../lib/token');
-const assertions = require('../../lib/assertions');
+
+const EarthdataLoginClient = require('../../lib/EarthdataLogin');
+
 const models = require('../../models');
-const {
-  createFakeJwtAuthToken,
-  fakeAccessTokenFactory
-} = require('../../lib/testUtils');
-const {
-  createJwtToken
-} = require('../../lib/token');
+const { fakeAccessTokenFactory } = require('../../lib/testUtils');
 
-process.env.TOKEN_SECRET = randomString();
-let accessTokenModel;
-let userModel;
-let jwtAuthToken;
+process.env.EARTHDATA_CLIENT_ID = randomId('edlID');
+process.env.EARTHDATA_CLIENT_PASSWORD = randomId('edlPW');
+process.env.DISTRIBUTION_REDIRECT_ENDPOINT = 'http://example.com';
+process.env.DISTRIBUTION_ENDPOINT = `https://${randomId('host')}/${randomId('path')}`;
+process.env.AccessTokensTable = randomId('tokenTable');
 
 
+process.env.TOKEN_SECRET = randomId('tokenSecret');
+let accessTokenModel;
+let authorizationUrl;
+
 // import the express app after setting the env variables
-const { app } = require('../../app');
+const { distributionApp } = require('../../app/distribution');
 
-test.before(async () => {
-  process.env.UsersTable = randomString();
-  userModel = new models.User();
-  await userModel.createTable();
 
-  process.env.AccessTokensTable = randomString();
-  accessTokenModel = new models.AccessToken();
+test.before(async () => {
+  accessTokenModel = new models.AccessToken('token');
   await accessTokenModel.createTable();
 
-  jwtAuthToken = await createFakeJwtAuthToken({ accessTokenModel, userModel });
+
+  const getAccessTokenResponse = {
```

Comment:
Any reason not to use the `fakeAccessTokenFactory()` here?

Reply:
Not sure what you mean, These are used for the 
`test('An s3credential request without access Token redirects to Oauth2 provider.'...`
here: https://github.com/nasa/cumulus/pull/881/files/98fe9dd86e944c54381793ab65091a56d242f629#diff-b3bf4d770a7ee12d7339b51e30fb86b0R100 . 
How would I use a fakeAccessTokenFactory?

Reply:
I could be misunderstanding, but it seems like `getAccessTokenResponse` is used right below as the stubbed return value for `EarthdataLoginClient.getAccessToken`. And the object is just populated with random values, so we could just as easily generate getAccessTokenResponse by doing 

`getAccessTokenResponse = fakeAccessTokenFactory()`

But I may be missing something

Reply:
I missed your point completely. You were talking about just for getAccessTokenResponse, I can create the stub from the factory. 0b3f64bef